src/Entities/Transaction.php

```diff
@@ -349,6 +349,16 @@ public function getMethod()
         return $this->getAttribute('method');
     }
 
+    /**
+     * @param array $value
+     *
+     * @return $this
+     */
+    public function setPaymentInstruction(array $data)
+    {
+        return $this->setAttribute('paymentInstruction', $data);
```

Comment:
I'm not sure if I should use a PaymentInstruction VO instead

Reply:
I guess the reason of fail is "write-only" property rather than VO.